test/Motor.Extensions.Hosting.NATS_IntegrationTest/NATSFixture.cs

```diff
@@ -1,22 +1,25 @@
 using System.Threading.Tasks;
-using TestContainers.Container.Abstractions;
-using TestContainers.Container.Abstractions.Hosting;
+using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Containers;
 using Xunit;
 
 namespace Motor.Extensions.Hosting.NATS_IntegrationTest;
 
 public class NATSFixture : IAsyncLifetime
 {
-    private readonly GenericContainer _container;
+    private readonly TestcontainersContainer _container;
+    private const int NATSPort = 4222;
 
     public NATSFixture()
     {
-        _container = new ContainerBuilder<NATSContainer>()
+        _container = new TestcontainersBuilder<TestcontainersContainer>()
+            .WithImage("nats:2.5")
```

Comment:
Also here, the newest version is 2.9.11. Do we need to update this?

Reply:
🫣